index.md

```diff
@@ -18,3 +18,13 @@ Our goals are:
 
 Today the project is still under [heavy development](https://github.com/kanidm/kanidm) to achieve these goals - We have many foundational parts in place, and many of the required security features, but it is still an Alpha, and should be
```

Comment:
unrelated, should we reword this to match the readme in main github?

Reply:
Yeah, I'll get that done now